apps/ledger-live-mobile/e2e/page/wallet/portfolio.page.ts

```diff
@@ -33,21 +38,25 @@ export default class PortfolioPage {
   addAccountCta = "add-account-cta";
   lastTransactionAmount = () => getElementById(this.transactionAmountId, 0);
 
-  navigateToSettings() {
-    return tapByElement(this.portfolioSettingsButton());
+  async navigateToSettings() {
+    await tapByElement(this.portfolioSettingsButton());
   }
 
-  openTransferMenu() {
-    return tapByElement(this.transferButton());
+  async openTransferMenu() {
+    await tapByElement(this.transferButton());
   }
 
-  navigateToSwapFromTransferMenu() {
-    // await scrollToText("Exchange crypto to crypto.", "transfer-scroll-list"); // FIXME: scroll not working
-    return tapByElement(this.swapTransferMenuButton());
+  async navigateToSwapFromTransferMenu() {
```

Comment:
same remark

Reply:
It's part of a path that should be tested.